src/twisted/trial/test/test_doctest.py

```diff
@@ -26,15 +26,15 @@ def test_id(self):
         for test in suite._tests:
             self.assertIn(idPrefix, itrial.ITestCase(test).id())
 
-    def test_basicTrialIntegration(self):
+    def test_basicTrialIntegration(self) -> None:
         """
         L{loadDoctests} loads all of the doctests in the given module.
         """
         loader = runner.TestLoader()
         suite = loader.loadDoctests(mockdoctest)
         self.assertEqual(7, suite.countTestCases())
 
-    def _testRun(self, suite):
+    def _testRun(self, suite: pyunit.TestSuite) -> None:
```

Comment:
just a drive-by comment.

I think that pyunit.TestSuite is ok

I am not sure we still need `twisted.trial.unittest.TestSuite`